tgui/packages/tgui/interfaces/SmartVend.tsx

```diff
@@ -19,10 +22,16 @@ type Data = {
 export const SmartVend = (props) => {
   const { act, data } = useBackend<Data>();
   const { contents = [] } = data;
+  const fallback = (
+    <Icon name="spinner" lineHeight="64px" size={3} spin color="gray" />
+  );
   return (
-    <Window width={440} height={550}>
+    <Window width={498} height={550}>
       <Window.Content scrollable>
         <Section
+          style={{
```

Comment:
prefer it this way so that if you add other ui elements they don't go out of focus while scrolling through the 5000 items in the fridge
```suggestion
      <Window.Content>
        <Section
          fill
          scrollable
          style={{
```